### ISSUE TITLE
Switch to battery app when charging

### ISSUE THREAD

**Issue by Kozova1** [JF002/InfiniTime#283]
The Pinetime Lite fork has this feature where it switches to the battery app when charging.  
This could be a very nice feature.

**Comment by pfeerick** [comment]
Yes please! I was going to suggest and loot at writing a simple vibrate when put on charge or taken off charge, i.e. like the pebble, but I think this is a much better idea!

**Comment by qbinary** [comment]
I would propose to just show a popup like when a notification is received. But to make that look good the line should fill faster. I made this #306 issue to address that.

**Comment by moriel5** [comment]
I personally prefer to see the battery percentage at all times on the home screen, in a similar fashion to how LineageOS shows it

**Comment by Riksu9000** [comment]
Would this address some problem, or what's the purpose of this?
